### DISCUSSION TITLE
Q: Create default user (admin) on initial migration

### DISCUSSION THREAD

**Discussion by LadIQe** [mikro-orm/mikro-orm#1545] [Q&A] [answered]
Hello,

I want to ask, if its possible to create default user, lets call him admin, on initial migration. I know that something like this is possible with Laravel, thats why Im asking.

Thank you

**Comment by B4nan** [comment, marked as answer]
You can have any query inside the migration, including insert query to add something to some table. Migrations are TS scripts but they contain just plain SQL queries. 

Soon we will also have support for seeding that will be maybe better suited for this - btw it will be also inspired by laravel, see #251 and #929.

**Reply by LadIQe** [reply]
ok thank you very much for reply.

Looking forward for this feature
